WalletWasabi.Fluent/ViewModels/MainViewModel.cs

```diff
@@ -128,6 +136,19 @@ public MainViewModel()
 				}
 			});
 
+		this.WhenAnyValue(x => x.MainScreen.CurrentPage)
+			.Subscribe(x =>
+			{
+				if (x is WalletViewModel walletViewModel)
+				{
+					CurrentWallet = walletViewModel;
+				}
+				else
+				{
+					CurrentWallet = null;
+				}
+			});
+
```

Comment:
Does it have any effect on anyithing?

Reply:
yes. allows the music control to animate away.

Reply:
If I remove I don't see any difference.

Reply:
contents of the music box will dissapear if you switch to a non - wallet page.